_posts/2024-12-27-documentos-estadoSII.md

```
@@ -0,0 +1,18 @@
+tags: [ riesgo, créditos , clientes, ejecutivos, operaciones ]
+comments: false
+author: Erick Burgos
+---
+
+## Ahora es más fácil verificar el estado del SII en los documentos de Advance
+
+Ahora, puedes consultar directamente el estado del SII en la sección de `Documentos` dentro de `Factoring`. Además, podrás filtrar los resultados por el estado del SII para realizar búsquedas más ágiles. Al colocar el puntero del mouse sobre cada estado, podrás obtener más detalles de manera rápida y sencilla.
+
+<video width="100%"  controls autoplay loop>
+  <source src="https://cdn.capitalexpress.cl/video/2024-12-27 11-06-46-EstadoSII-documentos.mp4" type="video/mp4">
+</video>
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)